### PR TITLE
Added cities / city select and add scooter functionality

### PR DESCRIPTION
**Pull request by JamesTTTT**
Added a city page filled with a list with all cities, a register city form as well as a filter-bar. Upon selecting a city a map is displayed with all scooters from city. Front-end is the same as figma design just a few functionalities missing. Scooter list is displayed under city data. You can now add new scooter to the api using add scooter form.

### CODE REVIEW

File: src/data/data.js [hunk 1]
```diff
@@ -61,26 +61,113 @@ export const scooterOverview = [
...
+  {
+    label: "Total Parking Zones",
+    data: "",
+  },
+  {
+    label: "Total Red Zones",
+    data: "",
+  },
+  {
+    label: "Total Green Zones",
```
Comment (FalkenDev) [inline src/data/data.js:97]: Geen zone är nu bonus parking zone

File: src/data/data.js [hunk 2]
```diff
@@ -61,26 +61,113 @@ export const scooterOverview = [
...
+  {
+    label: "Total Zones",
+    data: "",
+  },
+  {
+    label: "Total Parking Zones",
+    data: "",
+  },
+  {
+    label: "Total Red Zones",
```
Comment (FalkenDev) [inline src/data/data.js:93]: No parking zone

**Review by FalkenDev — APPROVED** [review]
Ser bra ut

File: src/data/data.js (hunk 1 above)
Comment (JamesTTTT) [reply src/data/data.js:97]: Ahhh tänkte väl

File: src/data/data.js (hunk 2 above)
Comment (JamesTTTT) [reply src/data/data.js:93]: Ska fixa

**Pull request merged**